### DISCUSSION TITLE
Update to classes.py for windows container

### DISCUSSION THREAD

**Discussion by cricboy007** [eshaan7/Flask-Shell2HTTP#15] [General]
I had to update line 131 in classes.py to **proc = subprocess.Popen(cmd, stdout=subprocess.PIPE, stderr=subprocess.PIPE, shell=True)** to avoid  _ERROR - [WinError 2] The system cannot find the file specified_ error in windows container.

**Comment by eshaan7** [comment]
Hi, @cricboy007. In your `cmd` are you trying to expand an environment variable or something similar ? ([reference](https://stackoverflow.com/a/36299483/10534470))
